api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineRunManagerLaunchTest.java

```diff
@@ -320,48 +322,56 @@ public void workerRunShouldUseParentRunCloudRegionWithParentRunIdPassedAsParamet
         assertThat(pipelineRun.getInstance().getCloudRegionId(), is(NON_DEFAULT_REGION_ID));
     }
 
-    private void mockToolManager() {
-        doReturn(tool).when(toolManager).loadByNameOrId(eq(IMAGE));
-        doReturn(tool).when(toolManager).resolveSymlinks(eq(IMAGE));
+    private void mockCloudRegions() {
+        mock(defaultAwsRegion);
+        mock(nonAllowedAwsRegion, false);
+        mock(notDefaultAwsRegion);
```

Comment:
For simple cases it is better to have two methods instead of one if you have a single boolean parameter. It helps code readability a bit because seeing `true` or `false` in the parameters doesn't really tell you anything.

```
mock(allowedRegion);
mock(notDefaultAwsRegion);
mockRestricted(notAllowedRegion);
```

Reply:
Done!